docs/source/_static/css/qb1-sphinx-rtd.css

```diff
@@ -321,16 +321,16 @@ h1, h2, .rst-content .toctree-wrapper p.caption, h3, h4, h5, h6, legend {
 }
 
 .wy-body-for-nav h1 {
-    font-size: 2.6rem;
+    font-size: 2.6rem !important;
     letter-spacing: -0.3px;
 }
 
 .wy-body-for-nav h2 {
-    font-size: 2.3rem;
+    font-size: 2rem;
 }
 
 .wy-body-for-nav h3 {
-    font-size: 2.1rem;
+    font-size: 2rem;
```

Comment:
Haven't checked how this looks like yet, but are we making h2 and h3 the same size then?

Reply:
Just got around and saw it in @stichbury local build https://stichbury.github.io/data/data_catalog.html#additional-settings-in-catalog-yml I'm wondering if we could indeed make h3 smaller. I see h4 are already quite big as well, maybe what about h3 1.8rem, h4 1.7rem?

Reply:
I made a mistake. h2 should be 2.3 still. Let me change.

I don't ever see any h4's on the page. I don't think we need to change that one.